### ISSUE TITLE
Logic for setting or not public voting for onsite final competion 

### ISSUE THREAD

**Issue by maroueneboubakri** [ctftime/ctftime.org#105]
Hi,

Why for some finals you don't put wight voting and you define the wight to zero and for some others the final is eligible to vote even-though there was quals for both of them ! in addition for the event you define to 0 it was the second final onsite.

For example:
https://ctftime.org/event/793
https://ctftime.org/event/819

And there is a lot of other example ! 

This is unfair and I hope consider changing this ! 

Regards
Maro

**Comment by Pharisaeus** [comment]
The first one is Canadian-only-individual CTF. You can't possibly expect it to have global rating points. 
Also a more general rule is `no uploaded scoreboard = no points`.

**Comment by maroueneboubakri** [comment]
The first one is a Tunisian CTF, the second one is Canadian CTF.  
Both are finals.
The issue is the qualifier for the Securinets CTF was online and available to all, we played the finals and didn't get any point since the weight was fixed to 0 without possible voting.

While the Candian CTF had the possibility to vote and since the scoreboard was not uploaded so it is normal to have 0 points

This is unfair and should be fixed.
